class/class.2.1/hello.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+const std::string FILENAME = "user.txt";
+
+class UserStats {
+public:
+    void load() {
```

Comment:
to improve testing capabilities it is better to path filename as an input argument and it will be a filed in UserStats class. `void load(const std::string& filename)` also for some cases it may be reasonable to return some bool result - in this case we can omit this while for example if our input file contains data in wrong format there should be a way to signal about it.